### ISSUE TITLE
Deleted files listed in index.js

### ISSUE THREAD

**Issue by jseppi** [mozfreddyb/eslint-plugin-scanjs-rules#10]
Looks like some of the rules files that were deleted in recent commits did not have their related lines removed from `index.js`, so this plugin throws an exception upon use.

```sh
$ eslint .
Cannot find module './lib/rules/assign_to_mozAudioChannel'
Error: Cannot find module './lib/rules/assign_to_mozAudioChannel'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/jamesseppi/CODE/federalist/node_modules/eslint-plugin-scanjs-rules/index.js:8:34)
```

Here's the line referenced in that exception trace: https://github.com/mozfreddyb/eslint-plugin-scanjs-rules/blob/master/index.js#L8 though I think there are more that will need to be removed as well.

**Comment by jordonbiondo** [comment]
@mozfreddyb can you expand a bit on what was changed/removed? Looks like some very large changes, my test suite is broken because of it, which is fine and temporary, but I'm wondering if I need to now be changing the way I use this package

**Comment by mozfreddyb** [comment]
Gah, sorry. I was relying on my editors "safe remove / refactor" feature and that didn't pan out so well. Fix coming soon.

**Comment by mozfreddyb** [comment]
@jordonbiondo Sorry, for doing you a disservice and breaking your tests.
I removed a lot of checks for proprietary JS/DOM extensions that were usable in Firefox OS to do privileged, potentially scary stuff.
With Firefox OS being gone for good, I figured I could remove all of those.
You can see the list of removed files in recent commits e.g., https://github.com/mozfreddyb/eslint-plugin-scanjs-rules/commit/f4c21db89a03100519abe0b4e3de24ed49f55db3 if you click on "99 files".

On a side note, you should probably use `eslint-config-scanjs`.
This plugin (eslint-plugin-scanjs) is a bucket of naive checks for things that are potentially dangerous. I.e., useful for warnings or manual testing, but not so useful for highlighting definite security risks.
Some things can be checked for in a less naive way and these are for example done in other libraries (that I also develope :))

This is why eslint-config-scanjs as an umbrella config.


(I should probably write that down on my blog or so, but I'm a bit short on time.)

**Comment by mozfreddyb** [comment]
To be fair, this is actually mentioned in the README file. Hooray me!

**Comment by jordonbiondo** [comment]
Thanks @mozfreddyb, really appreciate the speed of the fix, I'm still have a slight issue however which I've resolved for myself in #14.
